lib/chezmoi/boltpersistentstate.go

```diff
@@ -16,27 +16,27 @@ type BoltPersistentState struct {
 	db   *bolt.DB
 }
 
-// NewBoltPersistentState returns a new BoltPersistentState.
+// NewBoltPersistentState returns a new, unopened BoltPersistentState.
+// If you want to access a database b, open it with b.Open()
```

Comment:
Please drop the second line (`If you want...`): the statement is already implied by the inclusion of `unopened` in the first line.